redocly.yaml

```diff
@@ -32,20 +39,27 @@ theme:
     copyrightText: >-
       © 2013 - 2023 Ripple, All Rights Reserved.
     items:
-      - group: About
+      - group: Support
```

Comment:
The footer content should not be updated as opensource.ripple.com is a separate site and we don't want to add Ripple Enterprise product links in the footer. 
Just need the same list of links that we see on the site currently (screenshot attached)
<img width="1423" alt="Screenshot 2023-07-27 at 4 02 22 PM" src="https://github.com/ripple/opensource.ripple.com/assets/13425257/099ea45e-f263-4749-a210-89ea31eff627">

Reply:
Resolved